src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -532,13 +567,22 @@ class AddUserProfile extends Component {
     if (this.fieldsAreValid()) {
       this.setState({ showphone: false });
       if (!email.match(patt)) {
-        toast.error('Email is not valid,Please include @ followed by .com format');
+        toast.error('Email is not valid. Please include @ followed by .com format');
       } else {
         createUser(userData)
           .then(res => {
             if (res.data.warning) {
               toast.warn(res.data.warning);
             } else {
+              if (
```

Comment:
When "if within else" gets executed then toast.success will not execute. can we change this to else if and else to improve readability.